lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -321,6 +321,24 @@ def test_spoc_gradebook_pages(self):
         # Max number of student per page is one.  Patched setting MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 1
         self.assertEqual(len(response.mako_context['students']), 1)  # pylint: disable=no-member
 
+    def test_open_response_assessment_page(self):
+        """
+        Test that Open Responses is available only if course contains at least one ORA block
+        """
+        ora_section = '<li class="nav-item"><button type="button" class="btn-link" data-section="open_response_assessment">Open Responses</button></li>'  # pylint: disable=line-too-long
```

Comment:
Nit: could you define this variable across multiple lines for readability? If there's no way to do that without the linter complaining don't worry about it.

```
ora_section = (
    '<li class="nav-item">'
    ...
    '</li>'
)
```